ml-pipelines/README.md

```diff
@@ -0,0 +1,67 @@
+# ML Pipelines
+
+ML Pipelines is the Open Data Hub's pipeline solution for data scientists. It is built on top of the upstream [kfp-tekton](https://github.com/kubeflow/kfp-tekton) project. The Open Data Hub community has a fork of this upstream under the opendatahub org.
+
+1. [opendatahub-io/kfp-tekton](https://github.com/opendatahub-io/kfp-tekton)
+
+## Installation
+
+### Prerequisites
+
+1. The cluster needs to be OpenShift 4.9 or higher
+2. OpenShift Pipelines 1.7.2 or higher needs to be installed on the cluster
+3. The Open Data Hub operator needs to be installed
+4. Create a namespace for ML Pipelines to be installed in. ML Pipelines is installed in the `odh-applications` namespace.
```

Comment:
This line confuses me. Does the user need to create the odh-applications namespace?

Reply:
Whoops, I accidentally ate the `By default`